### PR TITLE
feat: toast notifications

### PR DESCRIPTION
**Pull request by tpluscode**
[https___view-builder.lndo.webm](https://user-images.githubusercontent.com/588128/196372797-98cc6df9-9288-41a2-a9dd-ac68c8aa803c.webm)

### CODE REVIEW

File: apps/api/lib/viewCollection.js [hunk 1]
```diff
@@ -32,15 +32,17 @@ export function construct({ client }) {
 
     // TODO: Construct generated from NodeShape
     return CONSTRUCT`
-      ?resource ${schema.identifier} ?id ; 
+      ?resource a ?type ;
+                ${schema.identifier} ?id ;
```
Comment (tpluscode) [inline apps/api/lib/viewCollection.js:36]: Slightly unrelated, without the RDF type, the UI cannot discover that a view is deletable

**Comment by cristianvasquez** [comment]
Thanks for the video! it's useful


I find the toast nice, but I imagine the interface could be cleaner if one uses an 'operation pending' indicator when triggering the action, followed by the toast. In this way, the user only sees one green toast for a more 'peaceful' experience

Maybe this one, https://shoelace.style/components/progress-bar?id=indeterminate, at the top or bottom, or something like that?

image

**Comment by cristianvasquez** [comment]
Unrelated to this pull request, I think the interface could benefit from some feedback to the user, indicating which is the current 'page'. I imagine this could be done by making the titles bold or using breadcrumbs.

If the user is looking at a view, the word 'view' is marked with a visual cue. 

image
(example exaggerating the boldness of 'view')

**Comment by tpluscode** [comment]
Thanks for your suggestions. See now

[https_view-builder.lndo.siteappviews.webm](https://user-images.githubusercontent.com/588128/196407693-dac0dc4e-e686-43e5-95fc-242fdc8e6d3f.webm)

**Review by cristianvasquez — APPROVED** [review]
looks ok to me!

**Pull request merged**